test/uk/gov/hmrc/plasticpackagingtax/returns/views/returns/ExportedPlasticWeightViewSpec.scala

```diff
@@ -129,7 +129,9 @@ class ExportedPlasticWeightViewSpec extends UnitViewSpec with Matchers {
 
       view must haveGovukGlobalErrorSummary
 
-      view must haveGovukFieldError("totalKg", "Enter an amount to continue")
+      view must haveGovukFieldError("totalKg",
+                                    messages("returns.exportedPlasticWeight.totalKg.empty.error")
```

Comment:
Should this remain as the literal value under test?  Same for the other specs...

My thoughts are that if the value associated with the key were to accidentally be changed, then the test would still pass.